src/plugins/validation/oas3/semantic-validators/responses.js

```diff
@@ -52,6 +52,13 @@ module.exports.validate = function({ resolvedSpec }, config) {
       } else {
         // default warnings for discouraged status code per IBM API Handbook
         for (const statusCode of statusCodes) {
+          if (!obj[statusCode].description) {
+            messages.addMessage(
+              path.concat([statusCode]),
+              `A ${statusCode} response should include a description.`,
```

Comment:
I didn't look at this close enough last time - because this check is an unconditional requirement and applies to every response, this message is slightly misleading. I think it should be changed to

```suggestion
              'All responses must include a description.',
```